client/src/components/game/gui/GUI.jsx

```diff
@@ -141,11 +147,21 @@ function GUI() {
                   onClick={() => togglePanel(Panels.Tasks)}
                   tooltipText={`${Utils.getTextDef("Tasks tooltip")} ( B )`}
                 />
+                {(inDungeon === true && (
+                <PanelButton
+                  icon={leaveDungeonIcon}
+                  onClick={() => togglePanel(Panels.LeaveDungeon)}
+                //   move to text defs
+                  tooltipText="Leave Dungeon"
+                />
```

Comment:
Looking at this syntax at 1 AM, makes me going wtf, never tocuhing JS ever this project

Reply:
honestly, yeah. I'll make it less confusing, they way it is now makes it harder to modify later. I'll change it to two separate if conditions